pkgs/by-name/xr/xrsh/package.nix

```
@@ -0,0 +1,62 @@
+{
+  stdenv,
+  fetchFromGitea,
+  makeWrapper,
+  writeShellApplication,
+  python3Packages,
+  lib,
+}:
+
+stdenv.mkDerivation rec {
```

You should use `finalAttrs` here instead of `rec` as that's better for overriding things (see [comparison](https://github.com/NixOS/nixpkgs/issues/315337#issuecomment-2138814370)):

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Then, in places where attributes are re-used, you'd need to prefix them with `finalAttrs`, for example:

```nix
a = finalAttrs.version;
```

The derivation doesn't appear to be using any internal attributes anyways so you can just remove `rec`, but this is something good to know in general.

---

@coderofsalvation Bump.

---

Resolved: Dropped `rec`.